### DISCUSSION TITLE
Where is new roadmap?

### DISCUSSION THREAD

**Discussion by 13ilya-old** [openssl/openssl#25114] [Q&A] [answered]
https://www.openssl.org/roadmap.html
The project used to have a roadmap, from which you could learn about the support period for each version and the innovations of new versions.

Where is it now and how can I find out the end of support for version 3.1, will there be a 3.4 LTS version, will it fully implement QUICK to support nginx and when will it be released?

**Comment by nhorman** [comment, marked as answer]
Current release planning is available here:
https://github.com/orgs/openssl/projects/12

I need to start updating planning board with LTS status and EOL dates

**Comment by Chealer** [comment]
There is no single document like [the old/original roadmap](https://web.archive.org/web/20160218065134/https://www.openssl.org/policies/roadmap.html), but there are both https://openssl-library.org/roadmap/ and [what would better correspond to what is usually called a roadmap](https://github.com/openssl/web/blob/master/roadmap.md). Unfortunately, the latter is outdated.

[🅭🄍](https://www.philippecloutier.com/Common+infrastructure+licensing#forum)
